controllers/mysql_container.go

```diff
@@ -318,19 +342,57 @@ func (r *MySQLClusterReconciler) makeV1InitContainer(cluster *mocov1beta2.MySQLC
 			}),
 	)
 
+	ok, err := r.isEnableLowerCaseTableNames(ctx, cluster)
+	if err != nil {
+		return nil, err
+	}
+	if ok {
+		c.WithArgs(constants.MocoInitLowerCaseTableNamesFlag)
+	}
+
 	updateContainerWithSecurityContext(c)
 	updateContainerWithOverwriteContainers(cluster, c)
 
-	var initContainers []*corev1ac.ContainerApplyConfiguration
-	initContainers = append(initContainers, c)
+	return c, nil
+}
 
-	spec := cluster.Spec.PodTemplate.Spec.DeepCopy()
-	for _, given := range spec.InitContainers {
-		ic := given
-		updateContainerWithSecurityContext(&ic)
-		initContainers = append(initContainers, &ic)
+func (r *MySQLClusterReconciler) makeInitContainerWithCopyMocoInitBin(cluster *mocov1beta2.MySQLCluster) *corev1ac.ContainerApplyConfiguration {
+	c := corev1ac.Container().
```

Comment:
Please set the default resource requests and limits.
We want make the mysql pods the Guaranteed QoS.